src/Post/Post.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CommentList from '../CommentList/CommentList';
+import User from '../User/User';
+import './Post.css';
+
+const Post = ({ post }) => {
+  const { user, comments } = post;
+
+  return (
+    <div className="post">
+      <h2 className="post__title">{post.title}</h2>
+      <p>{post.body}</p>
+      <User name={user.name} email={user.email} address={user.address} />
+      <CommentList comments={comments} />
+    </div>
+  );
+};
+
+Post.propTypes = {
+  post: PropTypes.shape({
+    comments: PropTypes.arrayOf({
+      comment: PropTypes.shape({
```

Comment:
create a separate variable for this `shape`, it's hard to read nested code
do this for all of your files

Reply:
for me it's more comfortable to see everything that object contains rather than going to file where this variable is and looking there